glusterd2/commands/volumes/commands.go

```diff
@@ -136,18 +136,6 @@ func (c *Command) Routes() route.Routes {
 			Version:     1,
 			RequestType: utils.GetTypeString((*api.VolStatedumpReq)(nil)),
 			HandlerFunc: volumeStatedumpHandler},
-		route.Route{
```

Comment:
if I want to get the volfiles for some debugging purpose, how can I do it?

Reply:
from mount log file

Reply:
question:
is it possible to generate a volfiles and return in response for this HTTP request?
will it be helpful for us later?

Reply:
Yes for client volfiles, need to get Volume info and generate the volfile on request. We can add that later if required. Since it generates fresh volfile on every request this may not be useful for debugging. Mount log file will be source of truth for actual volfile content used by glusterfs process.

Alternatively, We can dump the generated volfile to disk for debugging. (Note: Volfiles will be on disk only if a client(mount) connected to that glusterd2)